### PR TITLE
Add MeasuredValueType enum

### PR DESCRIPTION
**Pull request by tetienne**

### CODE REVIEW

File: pyoverkiz/enums/measured_value_type.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from enum import Enum, unique
+
+
+@unique
+class MeasuredValueType(str, Enum):
```
[the remark this answers is not in the document]
Comment (tetienne) [reply pyoverkiz/enums/measured_value_type.py:5]: Well OverkizAttribute contains the name of the attribute, `MeasuredValueType` here and not the available values.  If I follow you, we should do the same for DataType and  ProductType right?

[the remark this answers is not in the document]
Comment (iMicknl) [reply pyoverkiz/enums/measured_value_type.py:5]: You are right, I was a bit confused by everything being prefixed with core:. But this enum is the value indeed, not the key.

File: pyoverkiz/enums/measured_value_type.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    """See https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/68#issuecomment-1281474879"""
+
+    ABSOLUTE_VALUE = "core:AbsoluteValue"
+    ANGLE_IN_DEGREES = "core:AngleInDegrees"
+    ANGULAR_SPEED_IN_DEGREES_PER_SECOND = "core:AngularSpeedInDegreesPerSecond"
+    ELECTRICAL_ENERGY_IN_K_WH = "core:ElectricalEnergyInKWh"
```
Comment (iMicknl) [inline pyoverkiz/enums/measured_value_type.py:11]:
```suggestion
    ELECTRICAL_ENERGY_IN_KWH = "core:ElectricalEnergyInKWh"
```

File: pyoverkiz/enums/measured_value_type.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    ANGLE_IN_DEGREES = "core:AngleInDegrees"
+    ANGULAR_SPEED_IN_DEGREES_PER_SECOND = "core:AngularSpeedInDegreesPerSecond"
+    ELECTRICAL_ENERGY_IN_KWH = "core:ElectricalEnergyInKWh"
+    ELECTRICAL_ENERGY_IN_WH = "core:ElectricalEnergyInWh"
+    ELECTRICAL_POWER_IN_KW = "core:ElectricalPowerInKW"
+    ELECTRICAL_POWER_IN_W = "core:ElectricalPowerInW"
+    ELECTRIC_CURRENT_IN_AMPERE = "core:ElectricCurrentInAmpere"
+    ELECTRIC_CURRENT_IN_MILLI_AMPERE = "core:ElectricCurrentInMilliAmpere"
+    ENERGY_IN_CAL = "core:EnergyInCal"
+    ENERGY_INK_CAL = "core:EnergyInkCal"
```
Comment (iMicknl) [inline pyoverkiz/enums/measured_value_type.py:18]:
```suggestion
    ENERGY_IN_KCAL = "core:EnergyInkCal"
```

Would this be kilocalorie?

Comment (tetienne) [reply pyoverkiz/enums/measured_value_type.py:18]: I would say yes.

**Review by iMicknl — APPROVED** [review]

**Review by iMicknl — CHANGES_REQUESTED** [review]
Please add the file to https://github.com/iMicknl/python-overkiz-api/blob/main/pyoverkiz/enums/__init__.py as well, so we can import from .enums

**Comment by tetienne** [comment]
@iMicknl done

**Review by iMicknl — APPROVED** [review]

**Pull request merged**